apps/condo/domains/property/utils/testSchema/index.js

```diff
@@ -54,8 +54,8 @@ async function updateTestProperty (client, id, extraAttrs = {}) {
     return [obj, attrs]
 }
 
-async function makeClientWithProperty (includeFlat = false) {
-    const client = await makeClientWithRegisteredOrganization()
+async function makeClientWithProperty (includeFlat = false, clientWithOrganization) {
```

Comment:
It's not good to add a mandatory parameter after the not mandatory one. If you want to keep the order, please add the default value.

Reply:
it's not mandatory actually, you can freely skip it. it's only needed if you want to create BillingReceipt and all connected entities for same user

Reply:
removed that